pyzo/core/main.py

```diff
@@ -185,13 +186,11 @@ def _populate(self):
         # Create the default shell when returning to the event queue
         callLater(pyzo.shells.addShell)
         
-        # Create statusbar
-        if pyzo.config.view.showStatusbar:
-            pyzo.status = self.statusBar()
-        else:
-            pyzo.status = None
-            self.setStatusBar(None)
-        
+        # Create status bar
+        self.setStatusBar(StatusBar())
+        # show or hide
+        self.statusBar().setVisible(pyzo.config.view.showStatusbar)
```

Comment:
I was going to say that we need an entry in the default config, but then I saw we already have it. haha